src/Module/Special/HTTPException.php

```diff
@@ -41,14 +49,15 @@ class HTTPException
 	 * @param \Friendica\Network\HTTPException $e
 	 * @return array ['$title' => ..., '$description' => ...]
 	 */
-	private static function getVars(\Friendica\Network\HTTPException $e)
+	private function getVars(\Friendica\Network\HTTPException $e)
 	{
 		// Explanations are mostly taken from https://en.wikipedia.org/wiki/List_of_HTTP_status_codes
 		$vars = [
 			'$title' => $e->getDescription() ?: 'Error ' . $e->getCode(),
 			'$message' => $e->getMessage() ?: $e->getExplanation(),
 			'$back' => DI::l10n()->t('Go back'),
 			'$stack_trace' => DI::l10n()->t('Stack trace:'),
```

Comment:
Please remove the dependency to `DI` since you made the method dynamic.

Reply:
I knew it *g*
(I was just a little bit lazy and thought maybe you won't catch it :D ..)